lib/friendly_id/base.rb

```diff
@@ -196,6 +196,7 @@ module Base
     def friendly_id(base = nil, options = {}, &block)
       yield friendly_id_config if block_given?
       friendly_id_config.dependent = options.delete :dependent
+      # friendly_id_config.routes = (options.delete(:routes) || :friendly)
```

Comment:
Good catch :-)  I think I'd added it in there before I realized I could leverage the set.  Commented it out to test the set and then didn't delete it before committing :-)  I'll remove this in the next commit.